seed/data_importer/tasks.py

```diff
@@ -1701,6 +1740,10 @@ def pair_new_states(merged_property_views, merged_taxlot_views, sub_progress_key
 
     sub_progress_data.step('Pairing Data')
     view = next(chain(merged_property_views, merged_taxlot_views))
+    # Check if 'view' is an array, then
+    # use the first element, otherwise use 'view' as is
+    if isinstance(view, list):
+        view = view[0]
```

Comment:
Can you explain why this was added?

Reply:
This is no longer necessary. Thanks for reminding me to take this out.